components/automate-cli/pkg/verifyserver/server/api/v1/externalpostgresql.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) CheckExternalPostgresql(c *fiber.Ctx) {
+	externalPostgresqlRequest := new(models.ExternalPgRequest)
+	if err := c.BodyParser(&externalPostgresqlRequest); err != nil {
+		errString := fmt.Sprintf("External postgresql config request body parsing failed: %v", err.Error())
+		h.Logger.Error(fmt.Errorf(errString))
+		c.Next(&fiber.Error{Code: fiber.StatusBadRequest, Message: err.Error()})
+		return
+	}
+	pgConnection, _ := h.ExternalPostgresqlService.GetPgConnection(externalPostgresqlRequest)
```

Comment:
Are we not expecting any error here? (for gateway timeout)
If the error is returned, add an error log and return the error response.

Reply:
done